deploy/kubernetes/Makefile

```diff
@@ -1,6 +1,6 @@
 ALL=elastic-agent-standalone elastic-agent-managed
-BEAT_VERSION=$(shell head -n 1 ../../version/docs/version.asciidoc | cut -c 17- )
-BRANCH_VERSION=$(shell sed -n '2p' ../../version/docs/version.asciidoc | cut -c 14- )
+BEAT_VERSION=$(shell grep '^:stack-version:' ../../version/docs/version.asciidoc | cut -c 17- )
+BRANCH_VERSION=$(shell grep '^:doc-branch:' ../../version/docs/version.asciidoc | cut -c 14- )
```

Comment:
These changes were necessary because the previous implementation was a bit fragile.  It depended on specific variables being defined on specific line numbers in the `../../version/docs/version.asciidoc` file.  As such, adding any lines above those lines would break the parsing.